src/base.iced

```diff
@@ -428,9 +430,11 @@ class Base
 
   #------
 
-  generate_json : ({expire_in} = {}, cb) ->
+  generate_json : ({expire_in, version} = {}, cb) ->
     err = null
 
+    version or= constants.versions.sig
```

Comment:
can we change the name of this constant to `sig_v1` or something, for consistency/clarity?